### ISSUE TITLE
Sporadic failure in autodetection of when to use BigTIFF

### ISSUE THREAD

**Issue by mruffalo** [hubmapconsortium/ome-tiff-normalize#1]
We've seen some CellDIVE processing failures recently, which manifest as errors in SPRM -- this often looks like segmentation mask files that are missing one or more expected channels. This occurs due to issues in this normalization step, with the `bftools` code apparently missing some cases in its autodetection of when to use the BigTIFF output format.

Fix this by always writing as BigTIFF.

**Comment by mruffalo** [comment]
Example output and stack trace:
```
[OME-TIFF] -> reg002_S21100085_region_002_mask.ome.tiff [OME-TIFF]
Tile size = 5744 x 48
Reading IFDs
Populating metadata
        Converted 1/4 planes (25%)
        Converted 2/4 planes (50%)
        Converted 3/4 planes (75%)
Exception in thread "main" loci.formats.FormatException: File is too large; call setBigTiff(true)
        at loci.formats.out.TiffWriter.prepareToWriteImage(TiffWriter.java:391)
        at loci.formats.out.TiffWriter.saveBytes(TiffWriter.java:278)
        at loci.formats.out.OMETiffWriter.saveBytes(OMETiffWriter.java:218)
        at loci.formats.out.OMETiffWriter.saveBytes(OMETiffWriter.java:208)
        at loci.formats.tools.ImageConverter.convertTilePlane(ImageConverter.java:940)
        at loci.formats.tools.ImageConverter.convertPlane(ImageConverter.java:809)
        at loci.formats.tools.ImageConverter.testConvert(ImageConverter.java:742)
```

**Comment by mruffalo** [comment]
Fixed and released in `v1.2.1`.
